_licenses/mulanpsl-2.0.txt

```diff
@@ -0,0 +1,226 @@
+and any other combination which includes the Contribution. If you or your 
+Affiliates directly or indirectly institute patent litigation (including a 
+cross claim or counterclaim in a litigation) or other patent enforcement 
+activities against any individual or entity by alleging that the Software or 
+any Contribution in it infringes patents, then any patent license granted to 
+you under this License for the Software shall terminate as of the date such 
+litigation or activity is filed or taken.
+
+3. No Trademark License
+
+No trademark license is granted to use the trade names, trademarks, service 
+marks, or product names of Contributor, except as required to fulfill notice 
+requirements in section 4.
+
+4. Distribution Restriction
+
+You may distribute the Software in any medium with or without modification, 
+whether in source or executable forms, provided that you provide recipients 
+with a copy of this License and retain copyright, patent, trademark and 
+disclaimer statements in the Software.
+
+5. Disclaimer of Warranty and Limitation of Liability
+
+THE SOFTWARE AND CONTRIBUTION IN IT ARE PROVIDED WITHOUT WARRANTIES OF ANY 
+KIND, EITHER EXPRESS OR IMPLIED. IN NO EVENT SHALL ANY CONTRIBUTOR OR 
+COPYRIGHT HOLDER BE LIABLE TO YOU FOR ANY DAMAGES, INCLUDING, BUT NOT 
+LIMITED TO ANY DIRECT, OR INDIRECT, SPECIAL OR CONSEQUENTIAL DAMAGES ARISING 
+FROM YOUR USE OR INABILITY TO USE THE SOFTWARE OR THE CONTRIBUTION IN IT, NO 
+MATTER HOW IT’S CAUSED OR BASED ON WHICH LEGAL THEORY, EVEN IF ADVISED OF 
+THE POSSIBILITY OF SUCH DAMAGES.
+
+6. Language
+
+THIS LICENSE IS WRITTEN IN BOTH CHINESE AND ENGLISH, AND THE CHINESE VERSION 
+AND ENGLISH VERSION SHALL HAVE THE SAME LEGAL EFFECT. IN THE CASE OF 
+DIVERGENCE BETWEEN THE CHINESE AND ENGLISH VERSIONS, THE CHINESE VERSION 
+SHALL PREVAIL.
+
+END OF THE TERMS AND CONDITIONS
+
+How to Apply the Mulan Permissive Software License，Version 2 
+(Mulan PSL v2) to Your Software
+
+To apply the Mulan PSL v2 to your work, for easy identification by 
+recipients, you are suggested to complete following three steps:
+
+i. Fill in the blanks in following statement, including insert your software 
+name, the year of the first publication of your software, and your name 
+identified as the copyright owner;
+ii. Create a file named "LICENSE" which contains the whole context of this 
+License in the first directory of your software package;
+iii. Attach the statement to the appropriate annotated syntax at the 
+beginning of each source file.
+
+Copyright (c) [Year] [name of copyright holder]
+[Software Name] is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan 
+PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+         http://license.coscl.org.cn/MulanPSL2
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY 
+KIND, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO 
+NON-INFRINGEMENT
```

Comment:
There's a missing  coma here, compare to https://license.coscl.org.cn/MulanPSL2/ and https://github.com/spdx/license-list-data/blob/38ecfa995d4d472090e6730e4edf0296c6a12ed8/text/MulanPSL-2.0.txt

I also think this line could be folded with the next one and still be 78 characters or less.